packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -155,13 +153,6 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', '-Infinity')).to.equal(true);
       expect(nodeHasProperty(<div foo={0} />, 'foo', '-Infinity')).to.equal(false);
     });
-
-    it('should throw when un unquoted string is passed in', () => {
```

Comment:
Right - the problem is that we can't use spec-compliant selectors, because we're not selecting HTML - we're selecting from an RST, which has non-string values (HTML only has strings).